queries.sql

```diff
@@ -85,4 +85,10 @@ SELECT COUNT(*) FROM visits
 
 SELECT species.name FROM visits
   JOIN animals ON animals.id = animals_id JOIN vets ON vets.id = vets_id JOIN species ON species.id = species_id
-  WHERE vets.name = 'Maisy Smith' GROUP BY species.name ORDER BY COUNT(species.name) DESC LIMIT 1;
\ No newline at end of file
+  WHERE vets.name = 'Maisy Smith' GROUP BY species.name ORDER BY COUNT(species.name) DESC LIMIT 1;
+
+
```

Comment:
**[Optional]**
- [ ] As you know that this is a pair-programming activity but I cannot see any commit from your partner's side :fearful: Kindly ask your partner to make some contributions to this pull request :+1:

Reply:
I'm alone my coding partner is absent, I send a request to the student success